CMakeLists.txt

```diff
@@ -47,6 +47,7 @@ option (WITH_TLS "Enable Thread Local Storage (TLS) support" ON)
 option (WITH_UNWIND "Enable libunwind support" ON)
 
 cmake_dependent_option (WITH_GMOCK "Use Google Mock" ON WITH_GTEST OFF)
+set (WITH_FUZZING -fsanitize=fuzzer CACHE STRING "Fuzzing engine")
```

Comment:
The option is not meant to hold compiler flags or the like. It should be a symbolic name (e.g., fuzzer). Depending on the choice the corresponding flags or link libraries should be added. Something like

```cmake
if (WITH_FUZZING STREQUAL fuzzer)
  target_compile_options (fuzz_demangle PRIVATE -fsanitize=fuzzer)
endifif (WITH_FUZZING STREQUAL fuzzer)
```

Reply:
Ok @sergiud 

Should be as you expect now ;-)